apps/gimbal/src/main.cpp

```diff
@@ -56,9 +56,9 @@ void rxCANProcessor() {
 }
 
 int main() {
-  Utility::Logger::printf("\n\n");
-  Utility::Logger::printf("GIMBAL APPLICATION STARTED\n");
-  Utility::Logger::printf("=======================\n");
+  printf("\r\n\r\n");
```

Comment:
are the "\r\n" required for windows?

Reply:
Cuz i think they shouldn't be needed? https://os.mbed.com/docs/mbed-os/v6.8/apis/platform-options-and-config.html see the configurations for converting newlines. Should investigate if broken

Reply:
logic is all here:
https://github.com/ARMmbed/mbed-os/blob/4c4295225ec17f9485c61c72588a02f0d35d6b31/platform/source/mbed_board.c#L100
https://github.com/ARMmbed/mbed-os/blob/124afe79bb6412961e5981e7d75c0ecb3cbe604d/platform/source/mbed_retarget.cpp#L704

Reply:
hmm for uart printing it needs the `\r`, I'm not sure why that's not handled though